admin/sql/create_tables.sql

```diff
@@ -8,4 +8,15 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 
+CREATE TABLE listens (
+  id              SERIAL,
+  uid             VARCHAR NOT NULL,
+  date            TIMESTAMP,
+  timestamp       INT NOT NULL,
```

Comment:
My idea was,
`date` - The time when the listen was submitted
`timestamp` - Timestamp of the listen.